charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/csi-azuredisk-node-windows.yaml

```diff
@@ -91,7 +91,8 @@ spec:
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
             - "--metrics-address=0.0.0.0:{{ .Values.node.metricsPort }}"
-            - "--temp-use-driver-v2"
+            - "--temp-use-driver-v2={{ .Values.azuredisk.useV2Driver }}"
+            - "--is-node-plugin={{ .Values.azuredisk.useV2Driver }}"
```

Comment:
The `--is-node-plugin` flag is not understood by the upstream "v2" implementation and should be omitted.